test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/Crc32Tests.cs

```diff
@@ -40,5 +41,17 @@ public void Crc32_DifferentOutputForDifferentData()
             // Assert
             code1.Should().NotBe(code2);
         }
+
+        [Fact]
+        public void CalculateCrc_WithZeroBytes_CrcXorsToMagicNumber()
+        {
+            const uint zipCrc32MagicNumber = 0xdebb20e3;
```

Comment:
Please leave a comment stating the source of this "magic number".
e.g. - `// magic number as stated in the PKWare specification section 4.4.7 CRC-32`